{"name": "Form submission scenarios -- @1.1 ", "status": "failed", "statusDetails": {"message": "AssertionError: Message 'Thank You for your Message!555' not found on the page\n", "trace": "  File \"C:\\Users\\ki2ba\\AppData\\Roaming\\Python\\Python313\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ki2ba\\AppData\\Roaming\\Python\\Python313\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features\\steps\\contact_us_form_steps.py\", line 76, in verify_message\n    assert False, f\"Message '{expected_message}' not found on the page\"\n           ^^^^^\n"}, "steps": [{"name": "Given I am on the Contact Us form", "status": "passed", "start": 1752112613345, "stop": 1752112614930}, {"name": "When I fill in the \"first_name\" field with \"Joe\"", "status": "passed", "start": 1752112614931, "stop": 1752112615044}, {"name": "And I fill in the \"last_name\" field with \"Blogs\"", "status": "passed", "start": 1752112615045, "stop": 1752112615104}, {"name": "And I fill in the \"email\" field with \"joe_blogs@example.com\"", "status": "passed", "start": 1752112615105, "stop": 1752112615162}, {"name": "And I fill in the \"message\" field with \"Please can you contact me?\"", "status": "passed", "start": 1752112615162, "stop": 1752112615214}, {"name": "And I click on the \"SUBMIT\" button", "status": "passed", "start": 1752112615215, "stop": 1752112615677}, {"name": "Then I should see the message \"Thank You for your Message!555\"", "status": "failed", "statusDetails": {"message": "AssertionError: Message 'Thank You for your Message!555' not found on the page\n", "trace": "  File \"C:\\Users\\ki2ba\\AppData\\Roaming\\Python\\Python313\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ki2ba\\AppData\\Roaming\\Python\\Python313\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features\\steps\\contact_us_form_steps.py\", line 76, in verify_message\n    assert False, f\"Message '{expected_message}' not found on the page\"\n           ^^^^^\n"}, "start": 1752112615678, "stop": 1752112617699}], "attachments": [{"name": "Form submission scenarios -- @1.1 ", "source": "0e7dd179-02dc-4066-b65d-04526fe27729-attachment.png", "type": "image/png"}], "parameters": [{"name": "first_name", "value": "Joe"}, {"name": "last_name", "value": "Blogs"}, {"name": "email", "value": "joe_blogs@example.com"}, {"name": "message", "value": "Please can you contact me?"}, {"name": "button", "value": "SUBMIT"}, {"name": "expected_message", "value": "Thank You for your Message!555"}], "start": 1752112612744, "stop": 1752112620207, "uuid": "b84257ab-b2df-4fa5-9d13-1876d5182d76", "historyId": "6faf70435db598845c9c2609c31043f3", "fullName": "Contact Us form functionality: Form submission scenarios", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "regression"}, {"name": "feature", "value": "Contact Us form functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}